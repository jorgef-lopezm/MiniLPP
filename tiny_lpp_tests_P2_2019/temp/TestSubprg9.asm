global main
extern printf
section .data
true dd 1
false dd 0
const0 dd 1
fmt0 db "%d", 0

section .text
main:
push dword[const0]
push fmt0
call printf
add esp, 8
ret
