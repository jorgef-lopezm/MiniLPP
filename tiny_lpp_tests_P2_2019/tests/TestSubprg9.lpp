Inicio
	escriba 1
Fin
