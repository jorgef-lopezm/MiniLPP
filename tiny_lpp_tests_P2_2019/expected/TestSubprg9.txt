3541117197103593811591117162410351303